src/util/formatPriceToKoreanWon.ts

```diff
@@ -0,0 +1,5 @@
+const formatPriceToKoreanWon = (price: number) => {
+  return `${price.toLocaleString()}원`;
```

Comment:
KoreanWon이라면 조금 더 직관적으로 작성하면 좋겠네요! 🐫

```suggestion
  return `${price.toLocaleString('ko-KR')}원`;
```